lib/api/removeObject.js

```diff
@@ -31,8 +34,8 @@ function tripleIsValid (subject, predicate, object) {
   return true
 }
 
-function formatObject (object) {
-  if (object.startsWith('http')) {
+function formatObject (field, object) {
+  if (field === 'wasDerivedFrom' || field === 'type' || field === 'role') {
```

Comment:
Why change the way this was handled? Won't this lead to a big long list of field ifs?

Reply:
URIs do not need to start with http, so technically this was not correct.  These three fields expect URIs, so whatever is entered is a URI.